src/Admin/AdminInterface.php

```diff
@@ -659,29 +650,49 @@ public function getListMode();
      */
     // public function isCurrentRoute(string $name, ?string $adminCode = null): bool;
 
+    /**
+     * @param string $mode
+     */
+    public function setListMode($mode);
+
+    /**
+     * return the list mode.
+     *
+     * @return string
+     */
+    public function getListMode();
+
     /*
      * Configure buttons for an action
      */
-    // public function configureActionButtons(string $action, ?object $object = null): void;
+    // public function configureActionButtons(string $action, object $object = null): array;
 
     //TODO: uncomment this method for 4.0
     /*
      * Returns the result link for an object.
+     *
+     * @return string|null
      */
     //public function getSearchResultLink(object $object): ?string
 
 //    TODO: uncomment this method in 4.0
 //    /**
 //     * Setting to true will enable mosaic button for the admin screen.
 //     * Setting to false will hide mosaic button for the admin screen.
+//     *
+//     * @param bool $isShown
 //     */
-//    public function showMosaicButton(bool $isShown): void;
+//    public function showMosaicButton($isShown): bool;
```

Comment:
Are these changes fine? I mean, the argument's type declaration was removed and the return type was changed.

Reply:
Fixed